python/build.py

```diff
@@ -32,25 +32,58 @@
         if 'section' not in pack:
             pack['section'] = 'miscellaneous'
         if pack['section'] in packs:
-            packs[pack['section']] += [pack]   
+            packs[pack['section']] += [pack]
         else:
             packs[pack['section']] = [pack]
 
 pprint.pprint(packs)
 
+def _get_name(package):
```

Comment:
I don't think Is all this parsing is worth it. We should simply require an explicit `name` field. That would also allow proper capitalization for those crazy projects which claim to start with an uppercase lettter 😛 .

Reply:
OK, I simplified to: if you supply name we use that, if you don't we use the project name in `repo`.  Conversely, if you don't have a repo, you must supply `name:`.  I left pypi out of it.

Reply:
(oh, and this is maybe just laziness, I don't feel like going back and adding a `name:` field to all the entries.  If someone else wants to take that on and further simplify the parsing/get rid of it, they can do so)...

Reply:
I've added `name` to all packages in #137.